theano/tensor/nnet/ctc.py

```diff
@@ -175,13 +176,16 @@ def make_node(self, activations, labels, input_lengths):
         if t_input_lengths.ndim != 1:
             raise ValueError('input_lengths must have 1 dimension.')
 
+        openmp_enabled = 1 if gof.OpenMPOp(self).openmp is not None else 0
```

Comment:
I think it should be this line:

    openmp_enabled = 1 if self.openmp is not None else 0

self.openmp should exist and should be a bool.

Reply:
Yes, I was just not sure about the conversion and/or support for bool in C compilers, that is why I've used this expression with an int. Will fix that.